config/src/config.rs

```diff
@@ -236,10 +236,14 @@ pub struct Connections {
     ))]
     pub bucketing_ice_period: Duration,
 
-    /// Reject (tarpit) inbound connections coming from addresses in the same /16 IP range, so as
+    /// Reject (tarpit) inbound connections coming from addresses in the same /x IP range, so as
     /// to prevent sybil peers from monopolizing our inbound capacity (128 by default).
     pub reject_sybil_inbounds: bool,
 
+    /// Limit range to reject (tarpit) inbound connections coming from addresses in the same /x IP range, so as
+    /// to prevent sybil peers from monopolizing our inbound capacity (128 by default).
```

Comment:
The default changed :smile: 
```suggestion
    /// to prevent sybil peers from monopolizing our inbound capacity (128 by default).
```